src/App.js

```diff
@@ -1,41 +1,18 @@
 import "./App.css";
-import logo from "./logo.png";
-
+import {Header} from './component/Header'
+import { Route, Routes } from "react-router-dom";
+import Home from "./pages/Home";
+import {Footer} from "./component/Footer";
+import VideoListing from "./pages/VideoListing";

```

Comment:
You can create an index.js file and export all related files in it for both components and pages so that you can use barrel exports

Reply:
okay